meetings/2019-11-27.md

```diff
@@ -0,0 +1,90 @@
+﻿# Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-11-27
+
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=Pj2WOID9vNI
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/780
+
+## Present
+
+
+* Anna Henningsen @addaleax (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+

```

Comment:
I believe @tniessesn was there for most of the meeting.

Reply:
Er, I mean @tniessen ....

Reply:
``` suggestion
* Tobias Nießen @tniessen (TSC)
````